cluster-endpoints/src/grpc/gprc_accounts_streaming.rs

```diff
@@ -0,0 +1,215 @@
+use std::{collections::HashMap, time::Duration};
+
+use geyser_grpc_connector::GrpcSourceConfig;
+use itertools::Itertools;
+use solana_lite_rpc_core::{
+    commitment_utils::Commitment,
+    structures::{
+        account_data::{AccountData, AccountNotificationMessage, AccountStream},
+        account_filter::{AccountFilterType, AccountFilters, MemcmpFilterData},
+    },
+    AnyhowJoinHandle,
+};
+use solana_sdk::{account::Account, pubkey::Pubkey};
+use tokio::sync::broadcast;
+use tokio_stream::StreamExt;
+use yellowstone_grpc_client::GeyserGrpcClient;
+use yellowstone_grpc_proto::geyser::{
+    subscribe_request_filter_accounts_filter::Filter,
+    subscribe_request_filter_accounts_filter_memcmp::Data, subscribe_update::UpdateOneof,
+    SubscribeRequestFilterAccounts, SubscribeRequestFilterAccountsFilter,
+    SubscribeRequestFilterAccountsFilterMemcmp,
+};
+
+pub fn configure_account_streaming(
+    grpc_config: GrpcSourceConfig,
+    accounts_filters: AccountFilters,
+    account_stream_sx: tokio::sync::mpsc::UnboundedSender<AccountNotificationMessage>,
+) -> AnyhowJoinHandle {
+    tokio::spawn(async move {
+        loop {
+            // for now we can only be sure that there is one confirmed block per slot, for processed there can be multiple confirmed blocks
+            // So setting commitment to confirmed
+            // To do somehow make it processed
+            let commitment = yellowstone_grpc_proto::geyser::CommitmentLevel::Confirmed;
```

Comment:
nit: Mixing CommitmentLevel from solana-sdk and yellowstone proto used to be a nice source of weird bugs .. consider either 1. inline it or 2. use mapper between solana CommitmentLevel and yellowstone CommitmentLevel

Reply:
If there is an issue then it will be yellowstone_grpc issue.
yellowstone-grpc takes yellowstone_grpc_proto::geyser::CommitmentLevel as argument and we should provide it in same format and not rely on other code to give us required commitment.